Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.1.7
+
+RUN apt-get update -qq \
+  && apt-get install -y build-essential libpq-dev libqtwebkit-dev qt4-qmake
+
+WORKDIR /app
+ADD Gemfile /app/Gemfile
+ADD Gemfile.lock /app/Gemfile.lock
+RUN bundle install
+ADD . /app
+
+CMD ["/app/docker-runapp.sh"]
```

Comment:
Slight style nit pick, but please make sure that all files end with a newline. `docker-compose.yml` and `docker-runapp.sh` also need them.